addons/web/static/src/js/views/calendar/calendar_model.js

```diff
@@ -390,7 +390,7 @@ return AbstractModel.extend({
             monthNamesShort: moment.monthsShort(),
             dayNames: moment.weekdays(),
             dayNamesShort: moment.weekdaysShort(),
-            firstDay: moment().startOf('week').isoWeekday(),
+            firstDay: _t.database.parameters.week_start || moment().startOf('week').isoWeekday(),
```

Comment:
Is it useful to keep the moment() part? did you find a case where week_start was not defined?

Reply:
You are right after adding the damo data i just forgot to remove this code.